docs/versioned/integrations/aws.md

```diff
@@ -188,21 +180,22 @@ However, if you are streaming data to Iceberg, this will easily create a lot of
 Therefore, it is recommended to turn off the archive feature in Glue by setting `glue.skip-archive` to `true`.
 For more details, please read [Glue Quotas](https://docs.aws.amazon.com/general/latest/gr/glue.html) and the [UpdateTable API](https://docs.aws.amazon.com/glue/latest/webapi/API_UpdateTable.html).
 
-#### DynamoDB for Commit Locking
+#### Optimistic Locking
 
-Glue does not have a strong guarantee over concurrent updates to a table. 
-Although it throws `ConcurrentModificationException` when detecting two processes updating a table at the same time,
-there is no guarantee that one update would not clobber the other update.
-Therefore, [DynamoDB](https://aws.amazon.com/dynamodb) can be used for Glue, so that for every commit, 
-`GlueCatalog` first obtains a lock using a helper DynamoDB table and then try to safely modify the Glue table.
+By default, Iceberg uses Glue's optimistic locking for concurrent updates to a table.
+With optimistic locking, each table has a version id. 
+If you retrieve the table metadata, Iceberg records the version id of that table. 
+You can update the table, but only if the version id on the server side has not changed. 
+If there is a version mismatch, it means that someone else has modified the table before you did. 
+The update attempt fails, because you have a stale version of the table. 
+If this happens, Iceberg simply tries again by retrieving the table metadata and then tries to update it. 
+Optimistic locking prevents you from accidentally overwriting changes that were made by others. 
```

Comment:
I think we should not say it prevents people from doing something, we should say the optimistic locking guarantees atomic transaction of Iceberg tables in Glue.

Reply:
Thanks, I have made the changes.